tests/glibc/error.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+import sys
+from pyexpect import expect
+
+completed = subprocess.run([sys.argv[1]], check=True, capture_output=True)
```

Comment:
this could be just `check_output` instead of run

Reply:
This change has been made